monkestation/code/game/objects/effects/spawners/roomspawner.dm

```diff
@@ -42,6 +45,7 @@
 	dedicated_rooms = list(/datum/map_template/random_room/pubby_default_bar = 1,
 							/datum/map_template/random_room/pubby_japan_bar = 1,
 							/datum/map_template/random_room/pubby_pool_bar = 1,
-							/datum/map_template/random_room/pubby_hobo_bar = 1)
+							/datum/map_template/random_room/pubby_hobo_bar = 1,
+							/datum/map_template/random_room/pubby_yarr_bar = 1)
```

Comment:
This map feels a bit low effort.
There are too many pun puns, chairs are clipping through barricades, it's basically just a mess of stools and there isn't really any detail put into it.

Reply:
yeah I had 0 ideas for that one but I was asked to add a pirate bar by ook, any ideas of how I could improve it?

Reply:
I might just remove it for now and add it back in a future PR if I get any ideas.